pom.xml

```diff
@@ -50,7 +50,7 @@
         <dep.airlift.version>0.148</dep.airlift.version>
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.29</dep.slice.version>
-        <dep.aws-sdk.version>1.11.30</dep.aws-sdk.version>
+        <dep.aws-sdk.version>1.11.165</dep.aws-sdk.version>
```

Comment:
last bugfix to s3 was done in 1.11.162 version.
https://github.com/aws/aws-sdk-java/blob/master/CHANGELOG.md#111162-2017-07-13